src/cli/cmds/device.toit

```diff
@@ -409,10 +409,7 @@ extract-device fleet-device/DeviceFleet
 
   if format == "identity":
     if not identity-path:
-      fleet.write-identity-file
```

Comment:
It feels a little bit weird to me to talk about ids without knowing which server it is on. What do we store server-side when we create a new device identity?

Reply:
On the Artemis server we register the ID for the given organization.
On the broker we inform the server that the ID exists. We have a way to do this at a later point in time, and use this when we migrate a device from a broker to another.